ui/src/app/app-routing.tsx

```diff
@@ -98,14 +98,15 @@ export const AppRoutingComponent: React.FunctionComponent<RoutingProps> = ({onSi
           }} />}
       />
       <AppRoute path='/nih-callback' component={() => <HomepagePage routeData={{title: 'Homepage'}}/>} />
-    </ProtectedRoutes>
 
-    <ProtectedRoutes guards={[signInGuard, registrationGuard]}>
-      <AppRoute
+      <ProtectedRoutes guards={[signInGuard, registrationGuard]}>
+        <AppRoute
           path='/library'
           component={() => <WorkspaceLibraryPage routeData={{title: 'Workspace Library', minimizeChrome: false}}/>}
-      />
+        />
+      </ProtectedRoutes>
```

Comment:
I don't understand the bug with the previous approach, could you summarize the issue? Is it only possible to have one to-level ProtectedRoutes component? Probably a comment here would be useful to prevent a recurrence of this.